File: tests/led/led.c
/*
 *
 * Tests for the 'error' module.
 *
 */

#include <stdio.h>
#include <flipper.h>

#define DISCLOSE_ERRORS

int main(int argc, char *argv[]) {

	/* ~ Attach a Flipper device over USB. ~ */
	flipper.attach(FLIPPER_USB, "elroy");

	/* ~ Change the LED color. ~ */
	led.rgb(0, 0, 25);

	return 0;

}

File: api/rust/examples/led.rs
extern crate flipper;

use flipper::{
    Flipper,
    led::Led,
};

fn main() {
    let flipper = Flipper::attach().expect("should attach to Flipper");
    flipper.select_u2();
    let led = Led::new(&flipper);
    led.rgb(10, 0, 0);
}
